spec/models/case_contact_spec.rb

```diff
@@ -551,13 +551,13 @@
 
   describe "#contact_groups_with_types" do
     it "returns the groups with their associated case types" do
-      group1 = build_stubbed(:contact_type_group, name: "Family")
-      group2 = build_stubbed(:contact_type_group, name: "Health")
-      contact_type1 = build_stubbed(:contact_type, contact_type_group: group1, name: "Parent")
-      contact_type2 = build_stubbed(:contact_type, contact_type_group: group2, name: "Medical Professional")
-      contact_type3 = build_stubbed(:contact_type, contact_type_group: group2, name: "Other Therapist")
+      group1 = create(:contact_type_group, name: "Family")
+      group2 = create(:contact_type_group, name: "Health")
+      contact_type1 = create(:contact_type, contact_type_group: group1, name: "Parent")
+      contact_type2 = create(:contact_type, contact_type_group: group2, name: "Medical Professional")
+      contact_type3 = create(:contact_type, contact_type_group: group2, name: "Other Therapist")
       case_contact_types = [contact_type1, contact_type2, contact_type3]
-      case_contact = build_stubbed(:case_contact)
+      case_contact = create(:case_contact)
```

Comment:
These can be `build` instead of `create`. It would be a bit faster.

Reply:
wrong account :man_facepalming:

Reply:
Had to keep the create for the case_contact, else the spec fails. Changed the other ones.